dev/NavigationView/NavigationView.cpp

```diff
@@ -1326,13 +1359,105 @@ void NavigationView::ChangeSelection(const winrt::IInspectable& prevItem, const
     }
 }
 
+void NavigationView::UpdateIsChildSelected(winrt::IInspectable const& prevItem, winrt::IInspectable const& nextItem)
+{
+    // REMOVE CHECK ONCE TOP NAV FUNCTIONALITY IS IMPLEMENTED
+    if (IsTopNavigationView() || RootNodes().Size() == 0)
+    {
+        return;
+    }
+
+    auto lv = GetActiveListView();
+    auto viewModel = winrt::get_self<NavigationViewList>(lv)->ListViewModel();
+
+    if (lv && prevItem && !IsSettingsItem(prevItem))
+    {
+        winrt::TreeViewNode prevItemNode{ nullptr };
+
+        if (auto container = lv.ContainerFromItem(prevItem))
```

Comment:
This ContainerFromItem NodeFromContainer chaining is being used in multiple places, I actually just added a NodeFromItem helper in TreeViewList in PR #243. So I think  NodeFromItem should be added to MultiLevelListViewBase.